install/gcp/installer/forseti_server_installer.py

```diff
@@ -36,6 +35,7 @@ class ForsetiServerInstaller(ForsetiInstaller):
     access_target = None
     target_id = None
     user_can_grant_roles = True
+    firewall_rules_to_be_deleted = ["default-allow-icmp", "default-allow-internal", "default-allow-rdp", "default-allow-ssh"]
```

Comment:
This exceeds the 80-char limiit.

You can do:
```
firewall_rules_to_be_deleted = [
    'default-allow-icmp',
    'default-allow-internal',
    'default-allow-rdp',
    'default-allow-ssh',
]
```

Reply:
Done